CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.9.1 (TBD)
```

Comment:
I think some crates were already released at a `0.9.1` version (https://crates.io/crates/miden-objects), though not sure why.

Reply:
I can add the changelog entry here (or opened another PR with it) with the missed message of that merge 🤔

Reply:
Looks like this is the missing changelog entry https://github.com/0xMiden/miden-base/pull/1385